apps/community/src/components/NewMessagePageComponents/SendButton.tsx

```diff
@@ -37,6 +37,29 @@ export interface SendButtonProps extends ButtonProps {
   messageType: MessageType
 }
 
+const ABSTRACT_LENGTH = 153
+function getAbstract({ abstract, html }: { abstract?: string; html?: string }) {
+  if (abstract) return abstract
+  if (!html) return ''
+  const parser = new DOMParser()
+  const dom = parser.parseFromString(html, 'text/html')
+  const pStrList: string[] = []
```

Comment:
这里变量名用 `paragraphs` 意义比较明确
```suggestion
  const paragraphs: string[] = []
```